recordedit/recordEdit.app.js

```diff
@@ -263,14 +263,12 @@
                             throw response;
                         });
                     } else if (session) {
-                        var notAuthorizedMessage = "You are not authorized to Update entities.";
                         var notAuthorizedError = new Error(notAuthorizedMessage);
```

Comment:
You removed the `notAuthorizedMessage`, but it's still being used in this line. The following is not really related to your code, but since you're changing this piece of code it's better if you refactor it a bit.

Instead of creating a javascript `Error` object and manually changing the `code` attribute. You can use ERMrestjs. You can just say `new ERMrest.UnauthorizedError()` or `new ERMrest.ForbiddenError()`. You need to pass the `status`, and `message` to this function, but since we're not showing those attributes it might not be necessary. 

So please change all these error creations in this file to use ermrestjs error classes. You also missed the [error cases for create](https://github.com/informatics-isi-edu/chaise/blob/5e4ccaaf4ffb7baa0a34f8126d1d62f0528e3ccf/recordedit/recordEdit.app.js#L333,L346).